### ISSUE TITLE
Explain how PyPI deployments work in FAQ

### ISSUE THREAD

**Issue by treyhunner** [jazzband/roadies#63]
I am considering whether I could/should move [django-simple-history](https://github.com/treyhunner/django-simple-history) to jazzband and I realized I don't understand the expected process for maintaining PyPI deployments.

Would I assign one or more roadies as PyPI maintainers? Would I keep myself as the sole PyPI maintainer and deploy when someone else pinged me asking me to do so? Would we stop PyPI deployments?

I think this might be a good note to eventually add to the FAQ even if the answer is "it depends".

**Comment by jezdez** [comment]
So, this has changed since you opened this, I just launched the new "lead project members" feature, that will allow people to have more power over releases without risking malicious releases. Docs are now here: https://jazzband.co/about/releases.

The short version:

I've worked on a way for Jazzband members to join individual projects as "project members" and also to designate one or many of those to be "lead project members". The idea is to add a tiny bit more structure in the way Jazzband is organized right now and allow those project members to gain more powers that currently only the roadies have. As of right now I have not implemented creating Github teams for every project, but it's on my roadmap.

Once the project is set up for the process (currently a one-time-only process, but with the chance for fully automatic setup for new Jazzband projects) the "lead project members" are able to review package releases done via Travis-CI before they are released to PyPI.
